UMLEditor/UMLEditor/Classes/Diagram.cs

```diff
@@ -367,9 +365,10 @@ public string ListAttributes(string onClass)
     /// <exception cref="RelationshipNonexistentException">If the relationship does not exist</exception>
     public void DeleteRelationship(string sourceName, string destName)
     {
+        
         if (!RelationshipExists(sourceName, destName))
         {
-            throw new RelationshipNonexistentException($"Relationship {sourceName} => {destName} does not exist");
+            throw new RelationshipAlreadyExistsException($"Relationship {sourceName} => {destName} does not exists.");
```

Comment:
"exists" should be "exist"
"Relationship A => B does not exists" does not make sense